ActionSheet.android.js

```diff
@@ -87,19 +96,24 @@ class ActionGroup extends React.Component {
     );
 
     for (let i = startIndex; i < startIndex + length; i++) {
-      let color = '#212121';
-      if (i === destructiveButtonIndex) {
-        color = '#d32f2f';
-      }
-
+      const defaultColor = tintColor
+        ? tintColor
+        : (textStyle || {}).color || BLACK_87PC_TRANSPARENT;
+      const color = i === destructiveButtonIndex ? '#d32f2f' : defaultColor;
```

Comment:
you've placed some constants up top for colors, can you do the same for `#d32f2f`?